net-im/qtox/qtox-1.16.2.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils xdg-utils
+
+MY_P="qTox-${PV}"
+DESCRIPTION="Most feature-rich GUI for net-libs/tox using Qt5"
+HOMEPAGE="https://github.com/qTox/qTox"
+SRC_URI="https://github.com/qTox/qTox/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="notifications test X"
+
+S="${WORKDIR}/${MY_P}"
+
+RDEPEND="
+	dev-db/sqlcipher
+	dev-libs/libsodium
```

Comment:
libsodium uses subslots for SONAME changes, so please use `:=`. This will cause qtox to be automatically rebuilt whenever SONAME of libsodium changes.